webclient/src/containers/App/AppShell.tsx

```diff
@@ -23,15 +24,17 @@ class AppShell extends Component {
   render() {
     return (
       <Provider store={store}>
-        <CssBaseline />
-        <div className="AppShell" onContextMenu={this.handleContextMenu}>
-          <Router>
-            <Header />
+        <ToastProvider>
+          <CssBaseline />
```

Comment:
I think we still want `<CssBaseline />` to come directy after `<Provider store={store}>`.

Reply:
Seems reasonable.